### PR TITLE
[monorepo] Add VSCode settings

### PR DESCRIPTION
**Pull request by Alonski**
### Description

This unignores VSCode settings in the monorepo.
The settings are currently only related to ESLint.

### Related issues

#585 #573 

Fixes: #569

### CODE REVIEW

**Comment by snario** [comment]
`eslint` is not installed on some of those packages, yet.

**Comment by Alonski** [comment]
Hmm I'm not sure it matters but I'll update this tomorrow

**Comment by snario** [comment]
It matters because our editors will start forcing us to format any files in all of those folders because of `editor.formatOnSave`. This happened to me when I checked out the branch. It caused conflicts with the linters in those packages that I couldn't resolve (because of the `editor.formatOnSave`) until I changed branched and restarted VS Code.

**Comment by snario** [comment]
Could you also address / reference this issue here: https://github.com/statechannels/monorepo/issues/569

**Comment by tomclose** [comment]
It might be worth adding the `.vscode/settings` on a project-by-project level, as you convert to eslint. For example, #573 is currently waiting on vscode settings to be merged. It would be fine to enable `editor.formatOnSave` there, as it's been converted. Then we could get that merged.

File: packages/devtools/.eslintrc.js [hunk 1]
```diff
@@ -1,185 +1,185 @@
 const rules = {
```
Comment (snario) [inline packages/devtools/.eslintrc.js]: The changes here seem unrelated. I'm guessing they're because of my comment in the other PR? If so, I meant that we should avoid having literal object keys and quoted object keys. e.g., `{ "a": 1, b: 1 }`

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (snario) [inline .vscode/settings.json]: What's the point of having this root config and the package configs?

**Comment by snario** [comment]
I get that @tomclose mentioned the idea of having per-package `settings.json` files, but two questions:

1. Is that even necessary? It seems like `"eslint.workingDirectories"` handles that.

2. If it is necessary, why do we need a repo level config?

File: .vscode/settings.json (hunk 2 above)
Comment (andrewgordstewart) [reply .vscode/settings.json]: I believe that a root config is needed if I open the project from the root directory.

**Comment by Alonski** [comment]
@snario Some people like to open packages standalone and work inside them and not in the whole repo. 
The root level settings is for people working with the whole monorepo open.
Each package level settings is for people working with a lone package open,

File: packages/devtools/.eslintrc.js (hunk 1 above)
Comment (Alonski) [reply packages/devtools/.eslintrc.js]: This isn't related to the PR... But I ran format on this file and it changed :)
I can roll this back if its problematic to include

**Comment by Alonski** [comment]
@geoknee Done :)
